javascript/src/main/components/OfficeHours/OfficeHourForm.js

```diff
@@ -86,6 +94,32 @@ const OfficeHourForm = ({ createOfficeHour, updateOfficeHour, existingOfficeHour
                     />
```

Comment:
I would suggest making the code look something like this:
```jsx
{  createOfficeHour && <TutorAssignmentSelect
                    ...
                    /> 
}
{ updateOfficeHour && <TutorAssignmentDisplay tutorAssignment={tutorAssignment } /> }
```                   

The idea is that the we ONLY display the TutorAssignmentSelect drop down thingy if we are creating a new office  hour. 

We only display TutorAssignmentDisplay if we are editing an existing  office hour.

The TutorAssignmentDisplay will be a simple component that just takes a tutorAssignment object as a param, and displays it.  It doesn't need to be editable, or anything.ff